misk/src/main/kotlin/misk/logging/LogRateLimiter.kt

```diff
@@ -0,0 +1,48 @@
+package misk.logging
+
+import misk.Action
+import misk.web.actions.WebAction
+import misk.web.interceptors.LogRequestResponse
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+import javax.inject.Singleton
+import kotlin.reflect.KClass
+
+/**
+ * Holds rate limiters for logging success and error responses. There is a rate limiter for every
+ * action and service that calls it, for both success and error. The rate limiters are created
+ * according to the value set in [LogRequestResponse] annotation.
+ */
+@Singleton
+class LogRateLimiter @Inject constructor(
```

Comment:
This should be in the same package (or even same file) as the logging interceptors. The `logging` package shouldn't have a dependency on `web`.